service/storage_service.cc

```diff
@@ -1173,6 +1183,27 @@ future<> storage_service::raft_initialize_discovery_leader(raft::server& raft_se
             rtlogger.info("bootstrap: concurrent operation is detected, retrying.");
         }
     }
+
+    bool mark_sl = true;
+    while (mark_sl) {
+        auto sl_status = co_await _sys_ks.local().get_service_levels_migration_status();
+        if (sl_status) {
+            mark_sl = true;
+            break;
```

Comment:
is mark_sl unused?

Reply:
it was, I wanted to avoid while(true) but that solution wasn't good either. however, I've merged this change to gropu0 operation above